packages/common/source/types/index.ts

```diff
@@ -295,3 +275,40 @@ export interface Manifest {
     app: App;
     file?: string;
 }
+
+export interface Organization {
```

Comment:
You were supposed to derive doc comments based on the information given in the ticket.